app/assets/stylesheets/components/_filter-summary.scss

```diff
@@ -0,0 +1,51 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.app-c-filter-summary__remove-filter {
+  border: 1px solid $govuk-border-colour;
+  border-radius: govuk-em(5, 16px);
+  padding: govuk-spacing(2);
+  margin-right: govuk-spacing(2);
+  text-decoration: none;
+  text-wrap: nowrap;
```

Comment:
Not sure if this is what we want as then you get this on mobile:
<img width="433" alt="image" src="https://github.com/user-attachments/assets/ec045ed4-2768-4640-900b-02632362cbf8">

I wonder if a `flex`-based approach might work better here?